app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -51,11 +51,25 @@ class PatientSearchResultItemView(
     renderPatientNameAgeAndGender(searchQuery, model)
     renderPatientAddress(model.address)
     renderPatientDateOfBirth(model.dateOfBirth)
-    renderPatientPhoneNumber(model.phoneNumber)
+    val number = highlightPatientPhoneNumber(searchQuery, model)
+    renderPatientPhoneNumber(number)
     renderVisited(model.lastSeen)
     renderLastSeen(model.lastSeen, currentFacilityId)
   }
 
+  private fun highlightPatientPhoneNumber(searchQuery: String?, patientSearchResult: PatientSearchResultViewModel): PhoneNumber {
+    val canHighlightNumber = searchQuery?.let { searchQuery.isNotBlank() && patientSearchResult.phoneNumber.toString().contains(it) } == true
+    return if (canHighlightNumber) {
+      PhoneNumber.Highlighted(
+          patientNumber = patientSearchResult.phoneNumber.toString(),
```

Comment:
Since phone number can be nullable, calling `toString` might be problematic considering it will print literal `null` as text. You can perhaps use `orEmpty`.

Reply:
If phoneNumber is null, it wouldn't go inside the `if` condition at all.

Reply:
I think this comment was left in this commit where the null check was not explicitly set. So it can print out the `Plain` number as `null`.